.eslintrc.json

```diff
@@ -5,8 +5,18 @@
     }
   },
   "extends": [
-    "@lokalise/lokalise-frontend/core",
-    "@lokalise/lokalise-frontend/jest",
-    "@lokalise/lokalise-frontend/typescript"
+    "@lokalise/eslint-config-lokalise-frontend/core",
+    "@lokalise/eslint-config-lokalise-frontend/jest",
+    "@lokalise/eslint-config-lokalise-frontend/typescript"
+  ],
+  "overrides": [
+    {
+      "files": ["*.cjs"],
+      // Fall back to default parser for non-ts files
```

Comment:
❓ Was there an problem if this override isn't provided? If so let's raise a ticket to incorporate back into the eslint config.

Reply:
The issue was that I included only `src` folder in tsconfig so anything outside it wasn't getting handled by tsconfig and, by extension, by ts resolver / loader / whatever. In other words, eslint couldn't understand files outside src folder. I adjusted to include only `src` when compiling d.ts files though vite.